cli/units_cmd.go

```diff
@@ -38,6 +40,12 @@ func scanUnitsIntoConfig(cfg *config.Repository, quiet bool) error {
 		if err != nil {
 			return err
 		}
+
+		// Make sure that the file pointed to by cmdName is a regular executable file.
```

Comment:
It's probably better to move this comment to the declaration of `func checkRegularExecutableFile`, so its behavior is documented. No need for comment here, since the func name is descriptibe and will be documented after that move, so it's easy to figure out what it does.